proposals/NNNN-SDL-behavior-in-case of-Low-Voltage_mqueue.md

```diff
@@ -0,0 +1,67 @@
+# SDL behavior in case of LOW_VOLTAGE event
+
+* Proposal: [SDL-NNNN](NNNN-filename.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / Web / RPC]
+
+## Introduction
+
+This proposal defines SDL behavior when HMI requires SDL's services to be suspended due to a low voltage event.  
+A low voltage scenario is triggered when the battery voltage hits below a certain predefined threshold set by the system.  
+In case of such an event, the EMMC is turned off and all read/write operations are unavailable.  
+After the voltage level is restored all oprations are resumed.
+
+## Motivation
+
+Implement logic that will allow SDL to resume after battery charge is restored or to start up correctly in the next ingnition cycle if SDL was shut down due to LOW VOLTAGE event.  
+When battery voltage hits below the predefined threshold set by the system (e.g.7v) SDL will "freeze" all operations until it will be switched off or resumed.
+
+## Proposed solution
+
+Since SDL operations are to be halted including receiving and processing of normal RPC messages from HMI, it is proposed to setup a separate communication medium such as a message queue between HMI and SDL to exchange shutdown and wakeup signals.
+In case SDL receives "LOW_VOLTAGE" message via message queue, SDL must stop any read/write activities until SDL receives a "WAKE_UP" message via the message queue.  
```

Comment:
"In case SDL receives "LOW_VOLTAGE"" should be "In case SDL receives a "LOW_VOLTAGE""